libraries/AP_Airspeed/AP_Airspeed.h

```diff
@@ -9,6 +9,15 @@
 class AP_Airspeed_Backend;
 
 #define AIRSPEED_MAX_SENSORS 2
+#define AP_AIRSPEED_OPTIONS_DEFAULT          0       // No options on by default
+#define AP_AIRSPEED_FAILURE_TIME_PERIOD_MS   250     // Check the status every 250ms and increment counter if required
+#define AP_AIRSPEED_FAILURE_TIME_CLEAR_MS    60000   // Every 60 seconds to reset the counters
+#define AP_AIRSPEED_FAILURE_NUM_FAULTS       20      // 20 / 4 = 5 seconds of faults estimates in 60 secs
+#define AP_AIRSPEED_FAILURE_NUM_CLEAR        120     // 120 / 4 = 30 seconds of good readings estimates in 60 secs
+
+// Bitmask for airspeed options
+#define AP_AIRSPEED_FAILURE_MASK_BASIC       (1<<0)  // If set then use basic airspeed failure check
```

Comment:
These defines should all start with AP_AIRSPEED_OPTIONS_ and AP_AIRSPEED_OPTIONS_DEFAULT needs to be with it